nar/narinfo/parser.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/numtide/go-nix/hash"
 )
 
+//nolint:gocyclo
```

Comment:
what is is complaining about here?

Reply:
Stuttering - fixed in 887c56a07f1d4692f6725f05d250b57bd231e280.

Reply:
cyclomatic complexity 25 of func `Parse` is high (> 20). Probably fine to `nolint` that.